helper/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id("com.android.library")
+}
+
+android {
+    namespace = "io.github.libxposed"
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 24
+        targetSdk = 33
+
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        named("release") {
+            isMinifyEnabled = true
```

Comment:
This is a library. If you enable minify without correct proguard rules (currently empty), how can users use your library?